CruCentralCoast/Events/EventDetailsVC.swift

```diff
@@ -49,44 +57,113 @@ class EventDetailsVC: UIViewController {
         return .slide
     }
     
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        self.titleLabel.text = self.eventTitle
+        self.dateLabel.text = self.eventDate
+        self.descriptionLabel.text = self.eventSummary
+        self.imageView.image = self.eventImage
+//        self.locationButton.titleLabel?.text = self.eventViewableLocation
+        self.locationButton.setTitle(self.eventViewableLocation, for: .normal)
+//        print(self.locationButton.titleLabel?.text)
+    }
+    
     @IBAction func dismissDetail(_ sender: Any) {
         self.statusBarIsHidden = false
         self.topConstraint.constant = -20
+        closeButton.removeFromSuperview()
         self.dismiss(animated: true, completion: nil)
     }
     
+    
+    @IBAction func locationButtonPressed(_ sender: Any) {
+        guard let eventLocation = self.eventLocation else {
+            print("error1")
```

Comment:
Might want a more descriptive error message

Reply:
I'd try to only use print statements for debugging purposes and only push assertionFailures and console logging